### ISSUE TITLE
Highlight validator stashes

### ISSUE THREAD

**Issue by jacogr** [polkadot-js/apps#901]
With the change to displaying stashes in the validator list, the actual validators are not highlighted anymore. Current vs previous -

<img width="336" alt="Polkadot:Substrate Portal 2019-04-07 11-01-01" src="https://user-images.githubusercontent.com/1424473/55681258-8e251380-5924-11e9-8488-8323f91f6569.png">
<img width="324" alt="Polkadot:Substrate Portal 2019-04-07 11-00-47" src="https://user-images.githubusercontent.com/1424473/55681262-95e4b800-5924-11e9-9582-f1282e55a1a1.png">

Additionally in the actual dropdown for nominating, the same applies - actual current validators are not highlighted. (So the actual logic in the IdentityIcon should also check against the actual stashes and map back to try and find if this address belongs to either category)

https://github.com/polkadot-js/apps/blob/master/packages/ui-app/src/IdentityIcon.tsx#L61

Basically -

- attach `staking_bonded` in above
- if bonded & bonded.isSome, get bondId
- if bondId in `session_validators` (or address in there), highlight

(We could probably highlight with a different color as well if the user is active nominating or is an intention, yellow? This addition requires some work on https://github.com/polkadot-js/ui/blob/master/packages/ui-identicon/src/Identicon.tsx#L54 - optional, probably just "people in the queue" as opposed to everybody who is even nominating)

**Comment by jacogr** [comment]
Closed in https://github.com/polkadot-js/apps/commit/a878d0ca99563ae1d1b09466be3beda71b835f92

I f-ed up - the bloody branch creation didn't "take" in VSCode. (Need to re-add the branch protection and see wtf is up with people not being able to merge with that one)
